### ISSUE TITLE
Settable size of font in terminal

### ISSUE THREAD

**Issue by marusak** [cockpit-project/cockpit#11261]
Page:  Terminal

It is possible to set size of text in xterm.js
It would be nice to have, mostly as accessibility thing, where for some people size 12 of text can be rather small. It just needs design where this option should be placed.

Also see https://github.com/cockpit-project/cockpit/issues/11177
